features/other_a/invite_request.feature

```diff
@@ -62,6 +62,18 @@ Feature: Invite requests
       And I should not see "Sorry, you have no unsent invitations right now."
       And I should see "You have 2 open invitations and 0 that have been sent but not yet used."
 
+  Scenario: User can fail to send an invitation to an invalid email address
+
+    Given invitations are required
```

Comment:
Looking at `Scenario:  A user can manage their invitations` below, I think we can simplify this:

```
Given I am logged in as "user1"
  And "user1" has "1" invitation
  And I am on user1's manage invitations page
When I follow the link for "user1" first invite
  And I fill in etc
```